app.Makefile

```diff
@@ -28,14 +28,20 @@ $(shell echo '$(APP_PARAMETERS)' '$(APP_TEST_PARAMETERS)' \
     | docker run -i --entrypoint=/usr/bin/jq --rm $(APP_DEPLOYER_IMAGE) -s '.[0] * .[1]')
 endef
 
-KUBE_CONFIG ?= $(HOME)/.kube
-GCLOUD_CONFIG ?= $(HOME)/.config/gcloud
-EXTRA_DOCKER_PARAMS ?=
 
 .build/app: | .build
 	mkdir -p "$@"
 
 
+.build/app/dev: .build/var/MARKETPLACE_TOOLS_TAG \
+              | .build/app
+	$(call print_target)
+	docker run \
```

Comment:
Could you help me understand why we extract `scripts/dev` from the dev image, rather than running it directly?

Reply:
Chatted offline on this topic. The intent here is to not rely on the local filesystem implementation, as we move toward minimizing the build system requirements of this repository.

I would find it surprising that changes to `scripts/dev` are not reflected in `make app/install` and would prefer to run it from the local filesystem for that reason, but now that I know this I suppose it isn't of much consequence.

The target dependency semantics aren't really that clear to me (changes to `scripts/dev` or rebuilds of `gcr.io/cloud-marketplace-tools/k8s/dev` don't trigger `.build/app/dev` rebuilds), but I'm developing the opinion that we should spend effort on getting the next major release published and work off of docker images rather than figuring out clever make voodoo to make the development workflow simpler.

No action item here.